plyer/platforms/macosx/sms.py

```diff
@@ -0,0 +1,42 @@
+from subprocess import Popen, PIPE
+from plyer.facades import Sms as SMS
+from plyer.utils import whereis_exe
+
+
+class OSXSMS(SMS):
+    '''
+    Implementation of macOS' Messages API
+    '''
+
+    def _send(self, **kwargs):
+        '''
+        Will send `message` to `recipient` via Messages app
+
+        Currently only supports iMessage as macOS can send that standalone
+        In order to support SMS, a valid carrier-activated device must be
+        connected & configured to macOS
+            - activate this by passing in mode='SMS'
```

Comment:
Is something tested and that we can officially support?

Reply:
Yep, we can officially support it as long as the phone is correctly paired. You can test it and watch the messages turn green as it sends via SMS.